lib/internal/util/inspect.js

```diff
@@ -744,9 +749,11 @@ function formatValue(ctx, value, recurseTimes, typedArray) {
 
 function formatRaw(ctx, value, recurseTimes, typedArray) {
   let keys;
+  let privatePropertyPreviews;
   let protoProps;
   if (ctx.showHidden && (recurseTimes <= ctx.depth || ctx.depth === null)) {
     protoProps = [];
+    privatePropertyPreviews = getPrivatePropertyPreviews(ctx, value);
```

Comment:
Checking individually for `privatePropertyPreviews` requires a number of extra checks (everywhere, where `protoProps` is also checked). Instead, I suggest to rename `protoProps` and to add the private entries there. It is also logical to first show all regular properties, then to show all private members and then at the end all inherited ones.